content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -14,6 +14,7 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
 - [`metrics`](/concepts/components/prisma-client/metrics) since release [3.15.0](https://github.com/prisma/prisma/releases/tag/3.15.0) - [Submit feedback](https://github.com/prisma/prisma/issues/13579)
 - [`orderByNulls`](/concepts/components/prisma-client/filtering-and-sorting#sort-with-null-records-first-or-last) since release [4.1.0](https://github.com/prisma/prisma/releases/tag/4.1.0) - [Submit feedback](https://github.com/prisma/prisma/issues/14377)
 - [`tracing`](/concepts/components/prisma-client/opentelemetry-tracing) since release [4.2.0](https://github.com/prisma/prisma/releases/tag/4.2.0) - [Submit feedback](https://github.com/prisma/prisma/issues/14640)
+- [`multiSchema`](https://github.com/prisma/prisma/issues/...) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15077)
```

Comment:
Replace "documenation link" with link to comment describing the current state.

Reply:
```suggestion
- [`multiSchema`](https://github.com/prisma/prisma/issues/TODO) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15077)
```